includes/group-sites.php

```diff
@@ -1820,3 +1856,145 @@ function cboxol_load_theme_specific_fixes() {
 	}
 }
 add_action( 'wp_enqueue_scripts', 'cboxol_load_theme_specific_fixes' );
+
+/**
+ * Generate array of nav menu items.
+ *
+ * @return array $items
+ */
+function cboxol_get_nav_menu_items() {
+	$items = [];
+
+	$group_id = openlab_get_group_id_by_blog_id( get_current_blog_id() );
+	if ( ! $group_id ) {
+		return $items;
+	}
+
+	$group = groups_get_group( $group_id );
+	if ( ! $group->is_visible ) {
+		return $items;
+	}
+
+	$group_type = cboxol_get_group_group_type( $group_id );
+	if ( ! is_wp_error( $group_type ) ) {
+		$items[] = (object) [
+			'ID'               => 'group-profile-link',
+			'db_id'            => 0,
+			'object_id'        => $group_id,
+			'object'           => 'custom',
+			'title'            => '[ ' . $group_type->get_label( 'group_home' ) . ' ]',
+			'url'              => bp_get_group_permalink( $group ),
+			'slug'             => 'group-profile-link',
+			'type'             => 'custom',
+			'classes'          => [ 'group-profile-link' ],
+			'menu_item_parent' => 0,
+			'attr_title'       => '',
+			'target'           => '',
+			'xfn'              => ''
+		];
+	}
+
+	return $items;
+}
+
+/**
+ * Register meta box for CBOX OpenLab nav menu.
+ *
+ * @return void
+ */
+function cboxol_wp_nav_menu_meta_box() {
+	$is_group_site = (bool) openlab_get_group_id_by_blog_id( get_current_blog_id() );
+
+	// Only add meta box panel to group sites.
+	if ( ! $is_group_site ) {
+		return;
+	}
+
+	add_meta_box(
+		'add-buddypress-nav-menu',
```

Comment:
Looks like this bit of copy-paste was missed :-D

Reply:
(I just confirmed that the metabox only appears on nav-menus.php if we specify a unique `id`)

Reply:
Thanks for catching this. Resolved with 01f05d5.